examples/window-app/silabs/efr32/build_for_wifi_args.gni

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import("//build_overrides/chip.gni")
+import("${chip_root}/config/standalone/args.gni")
```

Comment:
Why is this needed?

Reply:
A tentative to integrate the common window app data model to the silabs folder.